lib/src/main/java/uk/gov/di/ipv/core/library/domain/JourneyRequest.java

```diff
@@ -1,13 +1,18 @@
 package uk.gov.di.ipv.core.library.domain;
 
 import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-@NoArgsConstructor
+import java.util.Map;
+
 @AllArgsConstructor
+@NoArgsConstructor
 @Data
+@Builder
 public class JourneyRequest {
     private String ipvSessionId;
     private String ipAddress;
+    private Map<String, String> pathParameters;
```

Comment:
The only path parameter which would be required across all of the lambdas is `journey` which would contain something like `/journey/cri/build-oauth-request/dcmaw` and would be passed in via the Step Function or API Gateway. So his could be just:
```java
private String journey;
private String ipvSessionId;
private String ipAddress;
...
```